mapper/census_test.go

```diff
@@ -140,6 +141,7 @@ func TestCreateCensusDatasetLandingPage(t *testing.T) {
 		So(page.Metadata.Title, ShouldEqual, datasetModel.Title)
 		So(page.Metadata.Description, ShouldEqual, datasetModel.Description)
 		So(page.DatasetLandingPage.Description, ShouldResemble, strings.Split(datasetModel.Description, "\n"))
+		So(page.IsNationalStatistic, ShouldResemble, datasetModel.NationalStatistic)
```

Comment:
You can use `ShouldBeTrue` instead. Can be more reliable test and is not dependent on the setup of your mocked model
```suggestion
		So(page.IsNationalStatistic, ShouldBeTrue)
```